### ISSUE TITLE
Allow suppressing highlighted text display

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#116]
Really minor issue, but still: right now it's impossible to avoid showing features highlighted in text, but in some cases it makes sense to hide them, for example when the document is large and number of features is small.

We have "force_weights" keyword argument that allows showing both all feature weights and highlighted features. It's possible to add another keyword argument "hide_text_highlightings", but that feels ugly.
Maybe it's better to solve it with "fields"? We have "targets" field that will mean "show features highlighted in text if possible, else weights". And let's add "targets_weights" and "targets_text_highlightings" fields: first will always shows weights, and second will always show features highlighted in text, and it will be valid to combine them with "targets" flag.
Does that make sense @kmike?

**Comment by kmike** [comment]
It makes sense to me. Will weight table include text features when targets_text_highlightings is False, but targets_weights is True?

**Comment by lopuhin** [comment]
Thanks! Yes, it will - all features will always be shown in weights table.
